app/src/main/java/com/codepath/apps/critterfinder/adapters/PetFavoritesAdapter.java

```diff
@@ -153,4 +151,10 @@ public void setOnItemClickListener(final OnItemClickListener mOnItemClickListene
         this.mOnItemClickListener = mOnItemClickListener;
     }
 
+    public void removePetAt(int position){
+       mPets.remove(position);
+        notifyItemRemoved(position);
+        notifyItemRangeChanged(position, mPets.size());
+    }
```

Comment:
I don't think we want to notify the adapter that the pet has been removed twice.

Suggested pattern that would be to have the delegate in the fragment call just these two lines:

```
pets.remove(position);
adapter.notifyItemRemoved(position);
```